webpack.config.babel.js

```diff
@@ -8,7 +8,7 @@ const env = process.env.NODE_ENV || 'development'
  * Base Configuration.
  */
 const config = {
-  devtool: 'cheap-eval-source-map',
+  devtool: env === 'production' ? '#eval' : '#eval-source-map',
```

Comment:
oh HA - that needs to be removed, good catch - i'll remove that

base config is fine so we don't have to define devtools in 2 places